eisbuk-admin/src/components/atoms/UserAttendance/UserAttendance.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction";
+import ListItemText from "@material-ui/core/ListItemText";
+import EisbukAvatar from "@/components/users/EisbukAvatar";
+import { Customer, BookingsMeta } from "eisbuk-shared";
+import Button from "@material-ui/core/Button";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import { ETheme } from "@/themes";
+import { markAttendance } from "@/store/actions/attendanceOperations";
+import { useDispatch } from "react-redux";
+
+type UserBooking = BookingsMeta & Pick<Customer, "certificateExpiration">;
+
+interface Props {
+  isAbsent: boolean;
```

Comment:
This should be `isAbsent?: boolean`. This way you won't have to assert non-null type in the calling of the component (like you did in the `AttendanceCard`) and IMO it's generally good to leave the `boolean` props optional in most cases (as undefined is falsy and can be easily be used instead of `false` and passing of the props when calling the component is easier).